web/src/components/storage/ProposalPage.test.jsx

```diff
@@ -76,6 +76,7 @@ const vdb = {
   size: 1e+6
 };
 
+// FIXME: needed to be reviewed/adapted after latest changes
```

Comment:
I don't get what this comment refers to.

Reply:
It is a left over. I added it at the very beginning of this sprint because I marked all tests as `skip`ed.